src/Components/CaseSummaryCard/CaseSummaryCard.tsx

```diff
@@ -23,18 +23,21 @@ export default function CaseSummaryCard(caseData: Case) {
             uri: caseData.imageUrl,
           }}
         />
-        <View style={styles.blurbContainer}>
-          <Text style={styles.blurbText}>{caseData.blurb}</Text>
-        </View>
-        <View style={styles.bottomContainer}>
+        <View style={styles.infoContainer}>
           <View style={styles.inLineInfo}>
-            <Text style={styles.subText}>
-              {formatDate(caseData.date)} • {caseData.lawFirm}
+            <Text style={[styles.subText, styles.lawFirmText]}>
+              {caseData.lawFirm}
+            </Text>
+            <Text style={[styles.subText, styles.dateText]}>
+              {' '}
+              • {formatDate(caseData.date)}
             </Text>
-            <View style={styles.threeDots}>
-              <ThreeDots />
-            </View>
           </View>
+          <Text style={styles.blurbText}>{caseData.blurb}</Text>
+        </View>
+        <View style={styles.bottomContainer}>
```

Comment:
Let's make this sticky/stuck!

Reply:
What are you referring to here? Are you talking about the "Learn more on case website" button on the `CaseSummaryScreen`?